src/format/scan.rs

```diff
@@ -221,7 +221,7 @@ pub(super) fn trim1(s: &str) -> &str {
 
 /// Consumes one colon char `:` if it is at the front of `s`.
 /// Always returns `Ok(s)`.
-pub(super) fn consume_colon_maybe(mut s: &str) -> ParseResult<&str> {
+pub(crate) fn consume_colon_maybe(mut s: &str) -> ParseResult<&str> {
```

Comment:
These two methods don't need to be public for the crate.

But maybe you should keep them like this, and put the `FromStr` implementation in `offset/fixed.rs`. It seems to be an exception to put them in this file.